packages/runtime/src/modules/DeciderModule.ts

```diff
@@ -139,24 +139,32 @@ export class DeciderModule extends RuntimeModule<DeciderModuleConfiguration> {
         const request = event.data.request;
 
         if (!containsItem(decideRequestItemParameters, isAcceptResponseConfig)) {
```

Comment:
please move this to a variable to make it more readable and maintainable

```suggestion
        const allItemsRejected = !containsItem(decideRequestItemParameters, isAcceptResponseConfig)
        if (allItemsRejected) {
```

Reply:
same for `if (!containsItem(decideRequestItemParameters, (element) => element === undefined)) {` a bit above with the variable name `allItemsAutomaticallyDecided` 

(I cant comment there)

Reply:
Alright, I'll also note a general refactoring of this module on my todo list. ;)

Reply:
I'd be in favor of doing this now as it will help the other reviews and this is already only a refactoring PR.

Reply:
I'll incorporate your comments now, but move the general refactoring to a later point in time after I've finish more pressing matters.

Reply:
Sure, that's exactly what I meant! :)

Reply:
Please don't forget to update the PR title to something a bit more generic ;)